packages/frontend/amp/components/Ad.tsx

```diff
@@ -123,5 +147,40 @@ export const Ad: React.SFC<{
                 config.usePrebid,
             )}
         />
+    );
+};
+
+export const Ad: React.SFC<{
+    edition: Edition;
```

Comment:
`edition` should be redundant now

Reply:
Same as the other place. It's not redundant as the targetting json still needs it.

Reply:
Quite right.  Ignore this comment.